styles/_base.scss

```diff
@@ -286,22 +291,43 @@ ul, ol {
 
 #milestone {
   // delete this rule in production
-  position: fixed;
-  bottom: -4px;
-  padding: 4px 132px 22px 8px;
-  font-size: 14px;
-  right: -6px;
-  border-radius: 4px;
-  border-width: 2px;
-  border-style: solid;
-  z-index: 10;
-
-  .milestone-inner {
-    position: relative;
-    top: 10px;
+  display: none;
+
+  .pre-release & {
+    display: block;
+    position: fixed;
+    bottom: -8px;
+    padding: 2px 132px 30px 8px;
+    font-size: 14px;
+    right: $scrollbar;
+    border-radius: 8px;
+    border-width: 2px;
+    border-style: solid;
+    z-index: 10;
+    @include chatOpeningTransition(right);
+
+    .milestone-inner {
+      position: relative;
+      top: 10px;
+    }
   }
+
+  .pre-release .chatOpen & {
+    right: 264px;
+  }
+
 }
 
 #doorbell-button {
-  right: 8px;
+  display: none;
+
+  .pre-release & {
+    display: block;
```

Comment:
The feedback button is also adjusted, and dependent on the pre-release class, so you can hide it and the milestone callout together.